packages/design-system/src/components/primitives/dnd/sortable-list.stories.tsx

```diff
@@ -63,43 +65,47 @@ export const SortableList = ({
     { id: "17", text: "Eighteenth" },
   ] as ItemData[]);
 
-  const [placement, setPalcement] = useState<{
-    index: number;
-    placementRect: Rect;
-  }>();
-
+  const [dropTarget, setDropTarget] = useState<DropTarget<true>>();
   const [dragItemId, setDragItemId] = useState<string>();
+  const rootRef = useRef<HTMLUListElement | null>(null);
 
-  const dropTargetHandlers = useDropTarget({
-    isDropTarget(element: HTMLElement) {
+  const useDropHandlers = useDrop<true>({
+    isDropTarget(element) {
       return element instanceof HTMLUListElement;
     },
-    onDropTargetChange(event) {
-      placementHandlers.handleTargetChange(event.target);
+    swapDropTarget(dropTarget) {
+      if (!dropTarget) {
```

Comment:
use early return
```suggestion
      if (dropTarget)  return dropTarget;
```

Reply:
This will not be the same as the current logic

Reply:
Ah, actually here it's the same, yes.